miniapps/parelag/MultilevelHcurlSolver.cpp

```diff
@@ -0,0 +1,739 @@
+// Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
```

Comment:
As done in some other miniapps, it may be helpful to comment at the beginning something like: "// We recommend viewing example 3 before viewing this miniapp."

Reply:
I'll add that both to the README and the .cpp.